docs/api-table.js

```diff
@@ -2,362 +2,371 @@ export const react = {
   Alert: {
     required: [
       [
-        'type',
+        "type",
```

Comment:
😱  I guess you have a misconfigured prettier or something here as well, adding double instead of single quotes? 😄

Reply:
...same thing in `bootExamples.js`, so I guess it goes for all `.js`-files.